docs/architecture-decisions/0003-implement-as-a-graphql-api.md

```diff
@@ -0,0 +1,81 @@
+# 3. Implement as a GraphQL API
```

Comment:
👍🏻 on the ADR conceptually, I have a few wording suggestions I'd like to implement (mostly moving a few bits around) but didn't want to do that yet in case others preferred it as-is (I wouldn't block moving forward exactly as written so my 👍🏻 is real)

Reply:
👍 on this ADR